docs/attribution/0001-analytics.rst

```diff
@@ -283,6 +283,52 @@ To use this method you will need to include ``datalayer-begincheckout-init.es6.j
     </a>
 
 
+Product Download
+~~~~~~~~~~~~~~~~
+
+.. Important::
+
+    Only Firefox and Pocket are currently supported. VPN support has not been added.
+
+We are using a the custom event `product_download` to track product downloads (Firefox and VPN). We are not using
+the default GA4 event file_download for a combination of reasons: it does not trigger for the Firefox file types,
+we would like to collect more information than is included with the default events, and we would like to treat
+product downloads as goals but not all file downloads are goals.
+
+Links to the app stores directly or through adjust are also treated as product downloads.
+
+.. Note::
+
+    Most apps listed in *appstores.py* are supported but you may still want to check that the URL
+    you are tracking is identified as valid in `isValidDownloadURL` and will be recognized by `getEventFromUrl`.
```

Comment:
Should these event and function names all be in double back-ticks for them to be formatted using `<code>`?
```suggestion
    you are tracking is identified as valid in ``isValidDownloadURL`` and will be recognized by ``getEventFromUrl``.
```